v0.1/index.md

```diff
@@ -6,32 +6,41 @@ title: Specification
 
 ## Definition
 
-This is an example of track specification:
+This is an example of a track specification:
 
-- Object: description of an object
-- Segment: a linear collection of objects
-- Track: a collection of segments
+- **Object**: an object of interest (such a cell) detected in a microscopy image
+- **Segment**: a linear, temporal collection of objects
+- **Track**: a collection of segments
 
 ## Scenarios
 
-### Simple track
+### A simple track
 
-![A simple track](SimpleTrack.png){:class="img-responsive"}
+The following shows an input consisting of 3 images.
 
+The case illustrated shows 3 objects identified in each frame. At this stage, there is no associations between objects across the frames.
 
+![1](images/SimpleTrack_objects.png){:class="img-responsive"}
```

Comment:
In https://github.com/pcmasuzzo/Tracks/blob/pcm_edit/v0.1/index.md I can't see the images rendered. Is there a capitalization issue in the image filename?